Lab_2.py

```diff
@@ -0,0 +1,80 @@
+import string
+import logging
+import sys
+from collections import Counter
+
+if __name__ == '__main__':

```

Comment:
It's unusual to have the `if __name__ == '__main__':` at the top.  Typically the layers go like I mentioned, module content, tests, then the if-main block.  However the code will work.  Just makes it easier to read when it's at the bottom as most people will look for module content at the top.